src/client/object/AppendForm.js

```diff
@@ -539,16 +539,22 @@ export default class AppendForm extends Component {
     const { intl, wObject, form } = this.props;
     const currentField = form.getFieldValue('currentField');
     const currentLocale = form.getFieldValue('currentLocale');
+    const filds = form.getFieldsValue();
     const filtered = wObject.fields.filter(
       f => f.locale === currentLocale && f.name === currentField,
     );
+    const triggerValue = trimStart(form.getFieldValue(currentField)).replace(/\s{2,}/g, ' ');
 
-    if (currentField === 'name') {
-      const triggerValue = trimStart(form.getFieldValue('name')).replace(/\s{2,}/g, ' ');
+    form.setFieldsValue({
+      [currentField]: triggerValue,
+    });
 
-      form.setFieldsValue({
-        [currentField]: triggerValue,
-      });
+    if (currentField === 'phone') {
+      if (filds.name) {
+        form.setFieldsValue({
+          name: trimStart(form.getFieldValue('name')).replace(/\s{2,}/g, ' '),
```

Comment:
Is 'form.getFieldValue('name)' the same as 'filds.name'? Better to use second one